main.js

```diff
@@ -222,8 +213,8 @@ async function _checkStatus({
   // get JSON RevocationList
   const {credentialSubject: rl} = slCredential;
 
-  if(rl.type !== 'RevocationList2021') {
-    throw new Error('Revocation list type must be "RevocationList2021".');
+  if(rl.type !== 'StatusList2021') {
+    throw new Error('Revocation list type must be "StatusList2021".');
   }
 
   // decode list from RL VC
```

Comment:
Should this be an `SL VC` instead? Can you check the rest of this file to make sure the right language is used?